### PR TITLE
Markdown format for extra.md

### PR DESCRIPTION
**Pull request by bitoiu**
Hi @Daniel-C-Dias @kaihj,

Please take my changes in. I hope these are ok, let me know in a code review.

:octocat::heart:

### CODE REVIEW

**Comment by bitoiu** [comment]
What do you think @Daniel-C-Dias?

**Comment by bitoiu** [comment]
Do I need to change something?

File: extra.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-random text to test da file
+## Extra topics
```
Comment (bitoiu) [inline extra.md:1]: I like the `##` heading. Is it ok?

Comment (Daniel-C-Dias) [reply extra.md:1]: I like it a lot ;)

**Comment by Daniel-C-Dias** [comment]
I haven't read it yet. Need to shovel some coal on the Internet

**Pull request merged**